style.css

```diff
@@ -0,0 +1,25 @@
+
+.container {
+    margin: 0 auto;
+    max-width: 1325px;    
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 15px;
+}
+
+.input-search {  
+     border: 3px solid #f8f8f8;
+     font-family: Muli-Regular;
+     color: #595959;
+     font-size: 16px;
+     background: url(images/shape.png);
+     background-repeat: no-repeat;
+     background-position: center left 10px;
+     padding-left: 40px;
+     height: 40px; 
+     width: 379px;
+     outline: none;
```

Comment:
You also need to add the `border-radius: 20px;`, border-radius, and border are not the same. Now I don't see the radius for the input 
![image](https://user-images.githubusercontent.com/5392202/94995795-9f2d0000-056e-11eb-9484-effc8fd37a23.png)